xbmc/cores/VideoPlayer/Process/ProcessInfo.cpp

```diff
@@ -655,6 +658,40 @@ int64_t CProcessInfo::GetMaxTime()
   return m_timeMax;
 }
 
+std::vector<EDL::Cut> CProcessInfo::GetCutList()
```

Comment:
The purpose of ProcessInfo is to hold information about the configuration of VP, not about the content. Adding cut list violates the original design.

Reply:
check out this: https://github.com/xbmc/xbmc/blob/master/xbmc/cores/VideoPlayer/VideoPlayer.cpp#L5054

Reply:
Okay. So the correct solution is to add methods for getting the cutlist and chapters directly to `CVideoPlayer`?

Reply:
... and to access it from `GUIInfoManager` via applplayer, like done with video stream info `g_application.GetAppPlayer().GetVideoStreamInfo(CURRENT_STREAM, video);`?

Reply:
I would keep it in CDataCache but not in CProcessInfo. VP can update DataCache directly.

Reply:
Is the issue only with processinfo and not with datacachecore? Is it okay to add cutlist and chaper methods to datacachecore (like I did) and to access the new vp methods from there?

Reply:
My idea was to remove polling of VP via methods like GetVideoStreamInfo completely. CDataCache is not a mirror of CProcessInfo but a cache for all info of VP. 
I introduced CDataCache to hold information about content: https://github.com/xbmc/xbmc/pull/5676

Reply:
Makes sense. Thanks for clarifying this. I will change the code accordingly.